src/main/lua/exasolrls/metadata_reader.lua

```diff
@@ -125,41 +125,69 @@ local function is_rls_metadata_table(table_id)
     return (table_id == "EXA_RLS_USERS") or (table_id == "EXA_ROLE_MAPPING") or (table_id == "EXA_GROUP_MEMBERS")
 end
 
-local function translate_table_metadata(schema_id)
+local function is_included_table(table_id, include_tables_lookup)
+    return include_tables_lookup[table_id]
+end
+
+local function create_lookup(include_tables)
+    local lookup = {}
+    if include_tables == nil then
+        setmetatable(lookup, {__index = function (_, _) return true end})
+    else
+        log.debug("Setting filter for metadata scan to the following tables: " .. table.concat(include_tables, ", "))
+        for _, table_id in ipairs(include_tables) do
+            lookup[table_id] = true
+        end
+    end
+    return lookup
+end
+
+local function translate_table_scan_results(schema_id, result, include_tables)
+    local tables = {}
+    local table_protection = {}
+    local include_tables_lookup = create_lookup(include_tables)
+    for i = 1, #result do
+        local table_id = result[i].TABLE_NAME
```

Comment:
Why do you call this `table_id` instead of `table_name`? Same for `schema_id`.

Reply:
To emphasize that those parameters must have SQL identifier quality.